File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/UserService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbUser;							  
import entity.PageResult;
import entity.Result;
/**
 * user服务层接口
 * @author jieweili
 *
 */
public interface UserService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbUser> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param user
     * @return
     */
    public Result saveUser(TbUser user);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbUser getUserById(Long id);

    /**
     * 修改对象
     * @param user
     * @return
     */
    Result updateUser(TbUser user);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeUser(Long[] idList);

    /**
     * 根据条件分页查询
     * @param user
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listUserByExample(TbUser user, Integer pageNum, Integer pageSize);
 
}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/userService.js
app.service('userService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../user/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../user/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeUserList = function (idList) {
        return $http.get('../user/removeUser.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../user/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateUser = function (user) {
        return $http.post('../user/updateUser.do', user)
    };

    //新增品牌
    this.saveUser = function (user) {
        return $http.post('../user/saveUser.do', user)
    };   
});
